src/components/Radio/stories.js

```diff
@@ -1,46 +1,178 @@
 import React from "react";
+import { Form, Field } from "react-final-form";
 import { storiesOf } from "@storybook/react";
+import PropTypes from "prop-types";
+
+import Button from "../Button/index";
+import GridCol from "../GridCol/index";
+import GridRow from "../GridRow/index";
+import Layout from "../Layout/index";
+import MultiChoice from "../MultiChoice/index";
 import Radio from ".";
 
+const onSubmit = values => {
+  console.log(JSON.stringify(values, 0, 2));
```

Comment:
boop

Reply:
🤫

Reply:
you can use "@storybook/addon-actions" for this